recipes/pro-mdnsd/all/conanfile.py

```diff
@@ -0,0 +1,79 @@
+import os
+import sys
+from conans import ConanFile, CMake, tools
+from conans.tools import download, unzip
+from conans.model.version import Version
+
+
+class mdnsdConan(ConanFile):
+    name = "pro-mdnsd"
+    license = "BSD-3-Clause"
+    exports_sources = [
+        "CMakeLists.txt",
+        "patches/**"
+    ]
+    homepage = "https://github.com/Pro/mdnsd"
+    url = "https://github.com/conan-io/conan-center-index"
+    description = "Improved version of Jeremie Miller's MDNS-SD implementation"
+    topics = ("dns", "daemon", "multicast", "embedded", "c")
+    settings = "os", "compiler", "build_type", "arch"
+    options = {
+        "fPIC": [True, False],
+        "shared": [True, False],
+        "compile_as_cpp": [True, False]
+    }
+    default_options = {
+        "fPIC": True,
+        "shared": False,
+        "compile_as_cpp": False
+    }
+    generators = "cmake", "cmake_find_package"
+
+    _cmake = None
+
+    @property
+    def _source_subfolder(self):
+        return "source_subfolder"
+
+    def configure(self):
+        if not self.options.compile_as_cpp:
+            del self.settings.compiler.libcxx
+            del self.settings.compiler.cppstd
+
+    def source(self):
+        tools.get(**self.conan_data["sources"][self.version])
+        folder_name = "mdnsd-" + self.version
+        os.rename(folder_name, self._source_subfolder)
+
+    def config_options(self):
+        if self.settings.os == "Windows":
+            del self.options.fPIC
+
+    def _configure_cmake(self):
+        if self._cmake:
+            return self._cmake
+        self._cmake = CMake(self)
+        self._cmake.verbose = True
```

Comment:
```suggestion
```

do not enable verbose by default

Reply:
https://github.com/conan-io/conan-center-index/pull/2009#discussion_r443839875

Reply:
remove it. for regular logs in CI it's not required. we enable it if it's required to analyze a problem for instance. ALL other recipes follow the same sense.